app/Models/Store.php

```diff
@@ -106,10 +106,10 @@ public function scopeDomainFilter(Builder $query, string|array $domains): Builde
         $first = array_shift($domains);
 
         return $query->where(function (Builder $subQuery) use ($first, $domains) {
-            $subQuery->whereJsonContains('domains', ['domain' => $first]);
+            $subQuery->whereJsonContains('domains', [['domain' => $first]]);
 
             foreach ($domains as $domain) {
-                $subQuery->orWhereJsonContains('domains', ['domain' => $domain]);
+                $subQuery->orWhereJsonContains('domains', [['domain' => $domain]]);
```

Comment:
Hey @elmodor this one doesn't feel right and caused test failures, 
see https://github.com/jez500/pricebuddy/actions/runs/14921703708/job/41918149143

Recommend running `lando phpcs & lando artisan test`

Reply:
Tests pass when this change is reverted https://github.com/jez500/pricebuddy/actions/runs/14921880100/job/41918647166

Reply:
Bummer, I guess we will have to distinguish between pgsql and msql connection type to do different queries then. 

I will have to runt he tests locally again and check

Reply:
Just FYI, I had to add this because PostgreSQL requires the object to be inside an array when querying json. When manually testing it also worked for mysql, but based on the test it might not work in all cases.